pkg/controllers/cloud/networkpolicy.go

```diff
@@ -790,6 +803,7 @@ type appliedToSecurityGroup struct {
 	securityGroupImpl
 	hasRules   bool
 	hasMembers bool
+	references map[string]bool
```

Comment:
references is generic. Can we change to addressGroupRef ??

Reply:
Done